xExtension-YouTube/configure.phtml

```diff
@@ -1,26 +1,29 @@
+<?php $this->loadConfigValues(); ?>
```

Comment:
I think it should be placed elsewhere… Maybe in the `init` method of the extension.php file?

Reply:
Yes, I agree. The init, or having the getters be lazy, and load their values when executed the first time ?

Reply:
Is that even required? I don't think so.

Reply:
To load them ? Yes, for the UI, as I converted `FreshRSS_Context::$user_conf->xxx` to the class getter (it was to simplify the code, because of the old confing conversion, to prevent code duplication).

Reply:
Okay, I tested in the `init`, but the settings in UI are "out of phase" (we always see the previous setting). I have the feeling they don't call the `init()` for the configuration (and it's the call in the `handleConfigureAction` which make them appear anyway).

Not sure why `loadConfigValues` is called in `handleConfigureAction` by the way.

Reply:
Ah right: `handleConfigureAction` is called both when showing the panel and when validating the form. So the call to `loadConfigValues` in this method is enough. However, it must be placed after saving the new values so it synchronizes internal values. In consequence, you just have to move `$this->loadConfigValues();` at the end of `handleConfigureAction` and you should be fine :+1:

Reply:
Fixed, thanks !